cloudformation/elk-stack.template

```diff
@@ -94,6 +94,10 @@
             "Description": "The KMS CMK to use to encrypt the EBS volume",
             "Type": "String"
         },
+        "MultimediaAccountId": {
+            "Description": "The AWS account ID for the Media Service",
+            "Type": "String"
```

Comment:
I think we try to keep account IDs out of github where possible (not that we do a very good job of this) so it might not make sense to provide a default.